contracts/Erc20Quest.sol

```diff
@@ -71,15 +70,24 @@ contract Erc20Quest is Quest {
         return redeemableTokenCount_ * rewardAmountInWeiOrTokenId;
     }
 
-    function withdrawRemainingTokens() public override onlyOwner {
-        super.withdrawRemainingTokens();
+    function withdrawRemainingTokens(address to_) public override onlyOwner {
+        super.withdrawRemainingTokens(to_);
+
+        uint unclaimedTokens = (receiptRedeemers() - reedemedTokens) * rewardAmountInWeiOrTokenId;
+        uint256 nonClaimableTokens = IERC20(rewardToken).balanceOf(address(this)) - protocolFee() - unclaimedTokens;
+        IERC20(rewardToken).safeTransfer(to_, nonClaimableTokens);
+    }
+
+    function receiptRedeemers() public view returns (uint256) {
+        return questFactoryContract.numberMintedForQuestId(questId);
+    }
 
-        uint256 nonClaimableTokens = (totalRedeemers - receiptRedeemers) * rewardAmountInWeiOrTokenId;
-        IERC20(rewardToken).safeTransfer(msg.sender, nonClaimableTokens);
+    function protocolFee() public view returns (uint256) {
+        return receiptRedeemers() * rewardAmountInWeiOrTokenId * questFee / 10_000;
     }
 
-    function withdrawFee() public {
-        IERC20(rewardToken).safeTransfer(protocolFeeRecipient, (receiptRedeemers * rewardAmountInWeiOrTokenId * questFee / 10_000));
+    function withdrawFee() public onlyStarted {
```

Comment:
I don't think it matters much (since `protocolFee()` is amount we can take), but should we check that quest is finished?

Or should we keep as is and withdraw as often as we please?

Reply:
@GarrettJMU what do you think? I think it's fine to withdraw the protocolFee whenever.